projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -28,58 +29,87 @@ import {
   filter,
   takeUntil,
   distinctUntilChanged,
+  catchError,
+  switchMap,
 } from 'rxjs/operators';
 
 import { Auth0ClientService } from './auth.client';
 import { WindowService } from './window';
 import { AbstractNavigator } from './abstract-navigator';
+import { connectableObservableDescriptor } from 'rxjs/internal/observable/ConnectableObservable';
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthService implements OnDestroy {
   private isLoadingSubject$ = new BehaviorSubject(true);
   private isAuthenticatedSubject$ = new BehaviorSubject(false);
+  private errorSubject$ = new ReplaySubject<Error>(1);
 
   // https://stackoverflow.com/a/41177163
   private ngUnsubscribe$ = new Subject();
 
+  /**
+   * Emits boolean values indicating the loading state of the SDK
```

Comment:
**nit**. Let's be consistent with the use of the full stop. @damieng would be proud.
```suggestion
   * Emits boolean values indicating the loading state of the SDK.
```